go/erasablekv/disable_backup_darwin.go

```diff
@@ -0,0 +1,24 @@
+// +build darwin
+
+package erasablekv
+
+import (
+	"path/filepath"
+
+	"github.com/keybase/client/go/libkb"
+	"github.com/pkg/xattr"
+	"golang.org/x/net/context"
+)
+
+const noBackup = "com.apple.metadata:com_apple_backup_excludeItem com.apple.backupd"
+
+func SetDisableBackup(ctx context.Context, g *libkb.GlobalContext, name string) {
+	path := filepath.Dir(name)
+	filename := filepath.Base(name)
+	// CrashPlan respects this metadata flag as does TimeMachine.
+	// https://support.crashplan.com/Troubleshooting/CrashPlan_And_OS_X_Metadata
+	err := xattr.Set(path, filename, []byte(noBackup))
+	if err != nil {
+		g.Log.CDebugf(ctx, "Unable to write xattr %s", filepath.Join(path, filename))
+	}
```

Comment:
Do we know of any cases where this fails today? If not, it might be better to let the error propagate. That way if some bug (or some new OS version) gets introduced in the future that makes everything fail, we're more likely to find out about it.

Reply:
I don't know of any failing cases today. I wasn't sure what would be best but decided just to log since it's a best effort feature and wouldn't want to break clients for something which may not be relevant to them